### ISSUE TITLE
This would be a neat command line tool

### ISSUE THREAD

**Issue by owainlewis** [garethrees/git-to-svn-guide#6]
Would be a neat command line tool to just map the git commands to their svn equivalent.

```
alias gsvn checkout = svn switch
alias gsvn pull = svn update
```

**Comment by garethrees** [comment]
Ha, yes it would! I've managed to avoid SVN since I wrote this, and I have little interest in going back to it. Would be a fun project to learn a little Go on, though!

**Comment by owainlewis** [comment]
Yeah would be easy as a CLI in Go. You should definitely learn it. Only takes a weekend to pick up 90% of it. It's the easiest language I've ever played with.
